syntax/import.go

```diff
@@ -38,6 +52,29 @@ func importLocalFile() rel.Value {
 	return importLocalFileVar
 }
 
+func findProjectRootDir(currentDir string) (string, error) {
```

Comment:
sysl does a very similar thing to find the .sysl directory, have a look at how it implements it @ https://github.com/anz-bank/sysl/blob/master/pkg/loader/loader.go#L105 .

Using the string functions and assuming "/" for paths will break on windows. Also, https://golangcode.com/check-if-a-file-exists/ is a bit of a better way to check for file existing, again this code will break on windows which allows case insensitive filename.